src/test/resources/solr/testCore/conf/schema.xml

```diff
@@ -145,7 +145,6 @@
        "resourcename": From SolrCell request param resource.name
    -->
    <field name="title" type="text_general" indexed="true" stored="true" multiValued="true"/>
-   <field name="uuid" type="string" indexed="true" stored="true"/>
```

Comment:
I would recommend leaving this in place for now. This is so that, until the LDPath expression baked into fedora is purged of the UUID field (c.f. https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-transform/src/main/resources/ldpath/default/nt_base_ldpath_program.txt), the default transformation from fcrepo will still contain a UUID value. If that value is sent to solr (without this configuration), the solr endpoint will throw an exception. You could make a note to remove this field after the corresponding PR is merged with the fcrepo-transform module.